aptos-move/block-executor/src/limit_processor.rs

```diff
@@ -221,14 +225,15 @@ impl<T: Transaction> BlockGasLimitProcessor<T> {
                 .block_gas_limit_type
                 .block_output_limit()
                 .map_or(false, |limit| accumulated_approx_output_size >= limit),
-            "[BlockSTM]: {} execution completed. {} out of {} txns committed",
+            "[BlockSTM]: {} execution completed. {} out of {} txns committed, took {}ms",
```

Comment:
optional: for the debugger use-case (and the output I saw) seems if we could somehow print more here (e.g. concurrency level) would be nice